<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>159</width>
          <height>244</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/icons/410.bmp</normaloff>:/images/icons/410.bmp</iconset>
        </attribute>
        <attribute name="label">
         <string>QListWidget Test</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0">
          <widget class="QToolButton" name="tBtnListClear">
           <property name="text">
            <string>tBtnListClear</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QToolButton" name="tBtnListInit">
           <property name="text">
            <string>tBtnListInit</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QToolButton" name="tBtnListInsert">
           <property name="text">
            <string>tBtnListInsert</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QToolButton" name="tBtnListAppend">
           <property name="text">
            <string>tBtnListAppend</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QToolButton" name="tBtnListDelete">
           <property name="text">
            <string>tBtnListDelete</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>159</width>
          <height>244</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget Test</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>159</width>
          <height>244</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTableWidget Test</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>QListWidget</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Current Item:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editCurrentItemText"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxListEditable">
             <property name="text">
              <string>Editable</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="tBtnSelectItem">
              <property name="text">
               <string>tBtnSelectItem</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::MenuButtonPopup</enum>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tBtnSelAll">
              <property name="text">
               <string>tBtnSelAll</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tBtnSelNone">
              <property name="text">
               <string>tBtnSelNone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tBtnSelInvs">
              <property name="text">
               <string>tBtnSelInvs</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/icons/check2.ico</normaloff>:/images/icons/check2.ico</iconset>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item 2</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/icons/check2.ico</normaloff>:/images/icons/check2.ico</iconset>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>QTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListInit"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actListInit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/128.bmp</normaloff>:/images/icons/128.bmp</iconset>
   </property>
   <property name="text">
    <string>Init List</string>
   </property>
   <property name="toolTip">
    <string>Init List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/delete1.bmp</normaloff>:/images/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>Clear List</string>
   </property>
   <property name="toolTip">
    <string>Clear List</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/424.bmp</normaloff>:/images/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>Insert Item</string>
   </property>
   <property name="toolTip">
    <string>Insert Item</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/316.bmp</normaloff>:/images/icons/316.bmp</iconset>
   </property>
   <property name="text">
    <string>Append Item</string>
   </property>
   <property name="toolTip">
    <string>Append Item</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/324.bmp</normaloff>:/images/icons/324.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
   <property name="toolTip">
    <string>Delete Item</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>Select Inverse</string>
   </property>
   <property name="toolTip">
    <string>Select Inverse</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/124.bmp</normaloff>:/images/icons/124.bmp</iconset>
   </property>
   <property name="text">
    <string>Select Item</string>
   </property>
   <property name="toolTip">
    <string>Select Item</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
